files/nviewjs/update.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packageManager": "github",
+  "name": "nviewjs",
+  "repo": "any-js/nviewjs",
+  "files": {
+    "basePath": "dist/",
+    "include": [
+      "nview.css", "nview.min.css",
+      "nview.js", "nview.min.js"
```

Comment:
Can you just leave the min files?
You can use this rule `*.min.*` to add them all